README.md

```diff
@@ -56,6 +56,10 @@ and see the results instantly without restarting your app or losing its state.
 
 [![Hot reload animation][]][Hot reload]
 
+### Developer Friendly
+
+Flutter works by using it’s own library of custom widgets designed for native platforms to give native like performance as compared to other cross platform technologies such as React, which uses an abstraction layer on top of the native api’s. This also means that if we have an update which depends on some material or Cupertino libraries, developers will be easily able to provide that same experience to users who weren’t able to update to the latest OS.
```

Comment:
`it's` -> `its`, `api's` -> `APIs`, `material` -> `Material Design` (iirc)

Please don't compare us to React. Flutter's voice is to speak accurately of itself and not mention other products.

Avoid "we". Flutter's voice is to talk of the product, not of the team.

I'm not really sure mentioning Cupertino here makes sense, it seems like a very specific thing to call out.

In general I'm not sure this change is really worth it. We already have content along these lines on flutter.dev which is the expected entrypoint for most readers.

Reply:
@Hixie  Your suggestions are worth noting and changes you mentioned above are also agreed, but i did find some difficulty in finding these subtle differences(which i added on readme) easily from the flutter.dev site thats why i think including them on readme may give a direct view to the developer while he browse the repo for the first time.

Reply:
That's fair, but at some level that's to be expected. There's never going to be a brief page that provides all the context, because there's too much context to be able to do it briefly. So we could mention that someone using Flutter will be able to provide the same experience for all versions of iOS, but then how about mentioning that on macOS desktop, we provide text field accessibility without using a private API, unlike Electron? There's so _many_ things that are worth mentioning that if you did mention them all, nobody would read it all, and so you'd guarantee some stuff would be missed anyway.